src/testhost.x86/DefaultEngineInvoker.cs

```diff
@@ -32,6 +33,8 @@ internal class DefaultEngineInvoker :
 
         private const string LogFileArgument = "--diag";
 
+        private const string DataCollectionPortArgument = "--datacollectionport";
```

Comment:
Not sure if it is a good idea to reuse testhost program logic for data collectors. Test host has complex sequences including app domain etc., datacollector doesn't need them.